.github/workflows/unit.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        python: ['3.6', '3.7', '3.8', '3.9', '3.10']
+        python: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
Could you drop 3.6 too?

I guess the latest ubuntu github runner doesn't include it anymore. Probably not worth pinning to a non-latest runner since we don't support 3.6 on GCF.

Reply:
Done